src/blocks/sandbox-image/edit.js

```diff
@@ -0,0 +1,260 @@
+	PanelRow,
+	SelectControl, // https://developer.wordpress.org/block-editor/reference-guides/components/select-control/
+	TextControl, // https://developer.wordpress.org/block-editor/reference-guides/components/text-control/
+	ToggleControl, // https://developer.wordpress.org/block-editor/reference-guides/components/toggle-control/
+} from '@wordpress/components';
+
+// Import our stuff.
+// import { Image } from '@bostonuniversity/block-imports';
+import { Image } from './imports/index.mjs';
+
+/**
+ * The edit function describes the structure of your block in the context of the
+ * editor. This represents what the editor will render when the block is used.
+ *
+ * @param  props
+ * @see https://developer.wordpress.org/block-editor/reference-guides/block-api/block-edit-save/#edit
+ *
+ * @return {Element} Element to render, in this case an image.
+ */
+export default function Edit( props ) {
+	const { attributes, setAttributes } = props;
+	// We could destructure the attributes, but for funsies I'm not going to this time...
+
+	// function StringToObjectConverter( userString ) {
+	// 	if ( ! userString ) {
+	// 		return undefined;
+	// 	}
+	// 	let userObject = {};
+
+	// 	try {
+	// 		// Wrap the object literal string in parentheses for eval() to parse it as an expression
+	// 		userObject = eval( '(' + userString + ')' );
+	// 		return userObject;
+	// 	} catch ( error ) {
+	// 		console.error( 'Error parsing userString:', error );
+	// 		return undefined;
+	// 	}
+	// }
+
+	// function StringToArrayConverter( userString ) {
+	// 	if ( ! userString ) {
+	// 		return undefined;
+	// 	}
+	// 	let userObject = {};
+
+	// 	try {
+	// 		userObject = eval( userString );
+	// 		console.log( userObject );
+	// 		console.log( typeof userObject );
+	// 		return userObject;
+	// 	} catch ( error ) {
+	// 		console.error( 'Error parsing userString:', error );
+	// 		return undefined;
+	// 	}
+	// }
+
+	const demo_mediaId = 381625; // 381625 or 381626 - FAILS, might need reload?
+	const demo_focalPoint = { x: 0.5, y: 0.5 };
+	const demo_labelsMediaPlaceholder = {
+		title: 'my custom title',
+		instructions: 'my custom instructions',
+	};
+	const demo_sourceSizes = [
+		{
+			srcset: 'thumbnail',
+			descriptor: '200w',
+			media: '(min-width: 600px)',
+		},
+		{
+			srcset: 'full',
+			descriptor: '2x',
+			media: '(min-width: 300px)',
+		},
+	];
+
+	// replaces wp image? https://github.com/WordPress/gutenberg/blob/c54fa0beb059a2e3b2d2f5a32f26ab47598be0b6/packages/block-library/src/image/edit.js
+	return (
+		<>
+			<div { ...useBlockProps() }>
+				<Image
+					// Component setup
+					canEditImage={ attributes.canEditImage }
+					debug={ true }
+					tag={ attributes.tag }
+					size={ attributes.size }
+					sourceSizes={ demo_sourceSizes }
+					altSource={ attributes.altSource }
+					mediaId={ demo_mediaId }
+					className={ attributes.className }
+					canEditFocalPoint={ attributes.canEditFocalPoint }
+					focalPoint={ demo_focalPoint }
```

Comment:
```suggestion
					focalPoint={ attributes.focalPoint } 
					onSelectFocalPoint={ handleFocalPoint }
```
I'd just use the attribute and move the default to block.json so the set focal point function can work.